src/ClimaLSM.jl

```diff
@@ -11,6 +11,8 @@ import .Parameters as LSMP
 include("SharedUtilities/Domains.jl")
```

Comment:
sure! I can try and do this

Reply:
I think I might wait on this...it will require changing a lot of source and test files (that extend e.g. ClimaLSM.function where function is defined in models.jl, or call them). Conceptually it makes sense to me that these would be modules (except maybe source, since it is so short). But I think there was some question of if in the long term, shared utilities would live in a separate repo. This would enable individual components to be developed in their own repos, without all the LSM tests, as long as they used/included the shared utilities repo. and then ClimaLSM would import the shared utilities (LSMs are also models) but also the individual components, and assemble them. Im not sure if this will happen, but im sort of inclined to wait on this until we're sure what the best course is?